pallets/attestation/src/benchmarking.rs

```diff
@@ -0,0 +1,79 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2021 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use delegation::{benchmarking::setup_delegations, Permissions};
+use frame_benchmarking::{benchmarks, impl_benchmark_test_suite};
+use frame_system::RawOrigin;
+use sp_core::sr25519;
+use sp_runtime::traits::Hash;
+use sp_std::num::NonZeroU32;
+
+use crate::*;
+
+const MAX_DEPTH: u32 = 10;
```

Comment:
We should have a max depth config in the Config trait and use that here

Reply:
Added in https://github.com/KILTprotocol/mashnet-node/pull/189/commits/fbb964e3c59d5d67066fd66039854b463a3454f6